src/app/components/LiveLabel/index.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { withKnobs } from '@storybook/addon-knobs';
+import { withServicesKnob } from '../../legacy/psammead/psammead-storybook-helpers/src';
+import LiveLabel from '../../legacy/psammead/psammead-live-label/src/index';
+import md from '../../legacy/psammead/psammead-live-label/README.md';
+import { StoryProps } from '../../models/types/storybook';
+import services from '../../../server/utilities/serviceConfigs';
+import Heading from '../Heading';
+
+interface Props extends StoryProps {
+  dir: string;
+  ariaHidden: boolean;
+  offScreenText: string;
+  liveText?: string;
+  text?: string;
+}
+
+const Wrapper = styled.div`
+  position: relative;
+`;
+
+export default {
+  title: 'New Components/Live Label',
+  decorators: [withKnobs(), withServicesKnob({ defaultService: 'pidgin' })],
+  parameters: {
+    docs: {
+      page: md,
+    },
+  },
+};
+
+export const WithLocalisedLiveText = ({ service, dir }: Props) => (
+  <LiveLabel
+    service={service}
+    dir={dir}
+    liveText={services[service].default.translations.media.liveLabel}
+  />
+);
+
+export const WithCustomOffscreenText = ({ service, dir }: Props) => (
+  <LiveLabel
+    service={service}
+    dir={dir}
+    ariaHidden
+    offScreenText="Watch Live"
+    liveText={services[service].default.translations.media.liveLabel}
+  />
+);
+
+export const WithChildren = ({ text: headline, service, dir }: Props) => (
+  <Wrapper>
+    <Heading level={3}>
+      <LiveLabel
```

Comment:
I am wondering (just to make it more like the previous story (in the now deleted https://github.com/bbc/simorgh/blob/3be17c140df6b0c76857fefa6f0ab83b487c7d0c/src/app/legacy/psammead/psammead-live-label/src/index.stories.jsx), and also to see what this might look like on our homepage, could we also use the InlineLink component from `src/app/components/InlineLink`?

```suggestion
    <Heading level={3}>
    <InlineLink text={headline} to="https://www.bbc.co.uk/ws/languages">
      <LiveLabel
```

Reply:
I have done it slightly differently because it made the live label disappear when inside the InlineLink, so I have put it after it and it looks good.

Reply:
The problem is - the Live label is not inside the link, but the header is... Can we try to make it work such that both are within the link?

Reply:
Ah on live the label isn't inside the link so didn't think it was necessary. I'll check the new designs.

Reply:
I think we still need to render the `InlineLink` as a child, rather than a sibling of the `LiveLabel` component, which is how it is done elsewhere in the application (e.g. on the FrontPage: https://github.com/bbc/simorgh/blob/latest/src/app/legacy/containers/StoryPromo/index.jsx#L233-L242)

e.g. 
```
<LiveLabel 
...
>
  <InlineLink>text</InlineLink>
</LiveLabel> 
```

The effect will be the same i.e. LIVE will not be inside the link, but it will have the correct text for the screenreader 🤞

Reply:
Done